XenModel/Actions/Updates/DownloadAndUnzipXenServerPatchAction.cs

```diff
@@ -203,23 +220,23 @@ private void ExtractFile()
                     iterator.Dispose();
 
                 if (downloadUpdate)
-                    File.Delete(zippedFileName);
+                    File.Delete(outputFileName);

```

Comment:
Apologies, I only saw this now: can we wrap this in a try-catch block too? An empty catch block will suffice:
```
if (downloadUpdate)
{
try { File.Delete(outputFileName); }
catch { }
}
```